osmaxx-py/osmaxx/tests/test_utils.py

```diff
@@ -0,0 +1,31 @@
+import string
+import random
+from django.test import TestCase
+import os
+from osmaxx.utils import PrivateSystemStorage
+
+
+class PrivateSystemStorageTestCase(TestCase):
+    def setUp(self):
+        random_directory_name_length = 10
+        self.random_directory_name = ''.join(
+            random.choice(string.ascii_lowercase + string.digits) for _ in range(random_directory_name_length)
```

Comment:
good point, didn't think of that. Since this is not a character, but a number, can we settle for `_throwaway`?